modules/modelSetup/BaseModelSetup.py

```diff
@@ -81,7 +81,7 @@ def calculate_loss(
             data: dict,
             config: TrainConfig,
     ) -> Tensor:
-        pass
+        ...
```

Comment:
Not that it does anything different but you probably shouldn't replace pass with ellipsis.

Reply:
Also done in https://github.com/Nerogar/OneTrainer/pull/212/commits/94506c6692da3eabdbba082e902b36747192feec.